packages/elements/src/react/sign-in/choose-session/choose-session.tsx

```diff
@@ -8,8 +8,8 @@ import {
   SignInActiveSessionContext,
   type SignInActiveSessionListItem,
   useSignInActiveSessionContext,
-  useSignInActiveSessionList,
   useSignInChooseSessionIsActive,
+  useSignInSessionList,
```

Comment:
Why not `useSignInAuthenticatedSessionList`? Do we want this to return revoked sessions?

Reply:
My change was mainly around not having the hook name tied to a certain statuses but rely on the return type 

This hook is mainly used internally for `@clerk/elements` components, so I see the reasoning of leaving it more explicit for our management